internal/testfiles/testfiles.go

```diff
@@ -0,0 +1,20 @@
+package testfiles
```

Comment:
Hm. Might want to think about the name of this one. The `ExcludeFiles()` method is not even exclusive to test files. Isn't the `fpath` package a more suitable home?

Reply:
ok. I'll move it to `fpath` package.